lib/src/zap/fabric/zap_fabric.c

```diff
@@ -2052,14 +2052,29 @@ static zap_err_t z_get_name(zap_ep_t ep, struct sockaddr *local_sa,
 			    struct sockaddr *remote_sa, socklen_t *sa_len)
 {
 	int ret = 0;
-	size_t sz = *sa_len;
+	size_t sz;
 	struct z_fi_ep *rep = (struct z_fi_ep *)ep;
 
 	if (!rep->fi_ep) {
 		return ZAP_ERR_NOT_CONNECTED;
 	}
+	if (0 == *sa_len) {
```

Comment:
Is this still a pull request that you want pulled in? Shouldn't this check if the sa_len is < the required address length?

Reply:
No, this doesn't need to be pulled in. The fix to `ldms_xprt_sockaddr` fixes the problem that zap_fabric can't be used with Munge. I'll close this pull request. Thanks.